frontend/package.json

```diff
@@ -78,6 +78,7 @@
     "react-modal": "3.x",
     "react-redux": "5.x",
     "react-router-dom": "4.x",
+    "react-router-scroll-4": "^1.0.0-beta.1",
```

Comment:
So this one is a fork: 
- fork: https://github.com/ytase/react-router-scroll  (this)
- orig: https://github.com/taion/react-router-scroll

But original doesn't support the behavior.  Looking it over.